File: Modul 4/Python/PRAK404-2410817120018-ArindaDitaPratiwi.py
while True:
    print("Pilih program")
    print("1.Penjumlahan")
    print("2.Pengurangan")
    print("3.Perkalian")
    print("4.Pembagian")
    print("5.Exit")
    pilihan = int(input("Masukkan Pilihan: "))
    if 1 <= pilihan <= 4:
        a = float(input("Masukkan nilai pertama: "))
        b = float(input("Masukkan nilai kedua: "))
        if pilihan == 1 :
            hasil = a + b
            print(f"Hasil penjumlahan antara {a:.2f} dengan {b:.2f} adalah {hasil:.2f}\n\n")
        elif pilihan == 2 :
            hasil = a - b
            print(f"Hasil pengurangan antara {a:.2f} dengan {b:.2f} adalah {hasil:.2f}\n\n")
        elif pilihan == 3 :
            hasil = a * b
            print(f"Hasil perkalian antara {a:.2f} dengan {b:.2f} adalah {hasil:.2f}\n\n")
        elif pilihan == 4 :
            hasil = a / b
            print(f"Hasil pembagian antara {a:.2f} dengan {b:.2f} adalah {hasil:.2f}\n\n")
    elif pilihan == 5 :
            print("Terima kasih, telah menggunakan kalkulator Arinda Dita Pratiwi\n\n")
            break
    else :
            print("Input anda salah, silahkan coba lagi\n\n")

File: Modul 4/C/PRAK404-2410817120018-ArindaDitaPratiwi.c
#include <stdio.h>
int main(){
    int pilihan;
    float a, b, hasil;
    while (1){
        printf("Pilihan program\n");
        printf("1.Penjumlahan\n");
        printf("2.Pengurangan\n");
        printf("3.Perkalian\n");
        printf("4.Pembagian\n");
        printf("5.Exit\n");
        printf("Masukkan Pilihan: ");
        scanf("%d", &pilihan);
    if(pilihan >=1 && pilihan <=4){
        printf("Masukkan nilai pertama: ");
        scanf("%f", &a);
        printf("Masukkan nilai kedua: ");
        scanf("%f", &b);
    switch(pilihan){
        case 1:
        hasil = a + b;
        printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f \n\n", a, b, hasil);
        break;
        case 2:
        hasil = a - b;
        printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f \n\n", a, b, hasil);
        break;
        case 3:
        hasil= a * b;
        printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f \n\n", a, b, hasil);
        break;
        case 4:
        hasil = a / b;
        printf("Hasil Pembagian antara %.2f dengan %.2f adalah %.2f \n\n", a, b, hasil);}}
        else if(pilihan == 5){
            printf("Terima kasih, telah menggunakan kalkulator ARINDA DITA PRATIWI\n\n"); 
            break;}
        else{printf("Input anda salah, silahkan coba lagi \n\n");}
        continue;
    return 0;}}